### PR TITLE
add docs, code and other standards document

### PR DESCRIPTION
**Pull request by galexrt**
Signed-off-by: Alexander Trost <galexrt@googlemail.com>

### CODE REVIEW

**Review by zalsader — APPROVED** [review]
LGTM, a few comments

File: docs/standards.md [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: Standards
+---
+
+## Documentation
+
+* Use Markdown Syntax.
+    * Depending on how the documentation is rendered (e.g., `mkdocs`), you can use certain [available Markdown extensions (Python library)](https://python-markdown.github.io/extensions/). Make sure to check which are available most commonly across our projects and documentations.
+    * For most projects/ documentations, the GitHub flavored Markdown syntax" is the "bare minimum" available. Make sure to use the available functionalities, e.g., better way to create tables.
```
[the remark this answers is not in the document]
Comment (galexrt) [reply docs/standards.md]: I have updated the text with a link and better wording.

**Comment by galexrt** [comment]
I have added a CI flow for commitlinting as I'm currently looking to add to all our repos.

**Comment by galexrt** [comment]
Todo for myself:

* [ ] Commit message format
* [ ] Add example structure of a "customer project"

**Review by ideepika — APPROVED** [review]
Lgtm, as is, added some optional suggestions!

**Comment by galexrt** [comment]
@zalsader @ideepika @gauravsitlani I have added more sections, please take a final review before I'll merge it.

**Review by zalsader — APPROVED** [review]

**Review by gauravsitlani — APPROVED** [review]

**Pull request merged**